SmartDeviceLink/SDLStreamingVideoLifecycleManager.m

```diff
@@ -593,6 +593,12 @@ - (void)sdl_didReceiveRegisterAppInterfaceResponse:(SDLRPCResponseNotification *
     if (resolution != nil) {
         self.videoScaleManager.displayViewportResolution = CGSizeMake(resolution.resolutionWidth.floatValue,
                                  resolution.resolutionHeight.floatValue);
+        // workaround for older high resolution displays that don't support scaling
+        NSString *make = registerResponse.vehicleType.make;
+        CGSize resolution = self.videoScaleManager.displayViewportResolution;
+        if (([make containsString:@"Ford"] || [make containsString:@"Lincoln"]) && (resolution.width > 800 || resolution.height > 800)) {
```

Comment:
This will occur for _all_ Ford and Lincoln head units with resolution width or height > 800px. Any other head unit released above that, this code will be triggered. It should be overwritten later if the head unit supports the video streaming capability, are you banking on that?

Reply:
Hi @joeljfischer ,
I will be the stand-in for Kujtim.
Regarding your query, this PR is just to address issues in legacy Ford IVIs. If IVI sends Video streaming capability, the new scale values should be used.